docs/architecture/sdk/index.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 3
+---
+
+# SDK Architecture
+
+Bitwarden provides a public Software Development Kit (SDK) for [Secrets Manager][sm] and an internal
+SDK for the Bitwarden [Password Manager][pm]. The SDK is written in Rust and provides bindings for
+multiple languages. The general end goal of the SDK is to own everything up to the presentational
+layers. This includes but is not limited to: API calls, data models, encryption, and business logic.
+
+<Bitwarden>We have collected a list of various learning resources on
+[Confluence](https://bitwarden.atlassian.net/wiki/spaces/DEV/pages/517898288/Rust+Learning+Resources).</Bitwarden>
```

Comment:
❓ Do we link to internal systems on this site? Not sure I have noticed that before. I guess if it's behind the Bitwarden veil we are good though.